project.mml

```diff
@@ -1720,25 +1720,13 @@ Layer:
       <<: *osm2pgsql
       table: |-
         (SELECT
-            way
+            way,
+            power
           FROM planet_osm_point
-          WHERE power = 'tower'
+          WHERE power = 'tower' OR power = 'pole'
```

Comment:
As said by @pnorman in https://github.com/gravitystorm/openstreetmap-carto/issues/1179#issue-52973210:
> It would be a performance gain if these were merged into one layer, ordered with poles before towers in the SQL.

I'm not sure why is that, but it looks like the order is important then.

Reply:
ok, done